src/AppBundle/EventListener/TodoNotifyListener.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Prooph\AppBundle\EventListener;
+
+use Prooph\ProophessorDo\Model\Todo\Command\MarkTodoAsExpired;
+use Prooph\ProophessorDo\Model\Todo\Command\RemindTodoAssignee;
+use Prooph\ProophessorDo\Model\Todo\TodoId;
+use Prooph\ProophessorDo\Model\Todo\TodoReminder;
+use Prooph\ProophessorDo\Projection\Todo\TodoFinder;
+use Prooph\ServiceBus\CommandBus;
+use Prooph\ServiceBus\Exception\CommandDispatchException;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\PostResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Class TodoNotifyListener
+ *
+ * @author Patrick Blom <info@patrick-blom.de>
+ */
+final class TodoNotifyListener implements EventSubscriberInterface
```

Comment:
EventSubscriberInterface is not needed here.

Reply:
see my [comment](https://github.com/prooph/proophessor-do-symfony/pull/12#issuecomment-325459672)

Reply:
Maybe I was not clear enough, and maybe I didn't unterstand everything well enough, but it looks to me like symfony event system is used and not the prooph event-bus, if so, why? With prooph's event-bus we don't need the EventSubscriberInterface from symfony.

Also the TodoNotifyListener is domain-related, and should not belong into an AppBundle.

Reply:
Thats correct the Symfony Event System is used here because, for me its was the right solution to attach something that listens on every request and check the read model for a change. 
It's definitely something you wont do in production because, as @codeliner mentioned earlier, it scales not very well. Normally you would add this kind of code in a separate worker, which asynchronously checks the timestamps and emits the commands. 

And that is in my opinion the reason why it has to be part of the AppBundle and not of the domain. Because its more like a workaround for educational purposes.

Maybe it will be useful to add comment inside the class to clarify that